force-app/main/default/lwc/geGiftBatch/__tests__/geGiftBatch.test.js

```diff
@@ -38,7 +38,7 @@ describe('ge-gift-batch', () => {
 
     it('should be in processing state', async () => {
         getGiftBatchView.mockResolvedValue(giftBatchViewJSON);
-        hasQueueableId.mockResolvedValue('DUMMY_QUEUEABLE_ID');
+        hasActiveRunningJob.mockResolvedValue(true);
```

Comment:
Let's add another test where `hasActiveRunningJob` is `false` and make sure the batch is *not* in a processing state, since that was the situation that caused this bug.

Reply:
added